frontend/src/components/TokenizeText.vue

```diff
@@ -32,22 +32,23 @@ function tokenizeText(): Token[] {
 function handleTokenClick(token: Token) {
   if (!token.isLink) return
   const toNode = createNode(token.content)
-  emit("linked-field:select", toNode)
+  emit("linked-field:clicked", toNode)
 }
 </script>
 
 <template>
-  <button
+  <!-- eslint-disable-next-line vuejs-accessibility/no-static-element-interactions -->
+  <span
     v-for="(token, idx) in tokenizeText()"
     :key="idx"
```

Comment:
hmm I am getting this eslint error when I remove it, but I am not sure if I have the latest eslint config to be honest 🫣
``` 
41:3  error  Elements in iteration expect to have 'v-bind:key' directives  vue/require-v-for-key
```

Reply:
Yes, we have to ignore the warning then. It makes sense to generally add a `key`, but I'm not sure, what that would be here? vue can handle this (react cannot):

> Without keys, Vue uses an algorithm that minimizes element movement and tries to patch/reuse elements of the same type in-place as much as possible. With keys, it will reorder elements based on the order change of keys, and elements with keys that are no longer present will always be removed / destroyed.

Using`index` is always wrong as it is never an identifier for a list item. As soon as you start changing the order or removing or adding elements other than the last one, this can lead to problems.

Reply:
thanks for explaining 🫶  I will remove the index and suppress the warning as I don't think there is any unique identifier